libsolidity/lsp/LanguageServer.cpp

```diff
@@ -300,26 +302,26 @@ void LanguageServer::compileAndUpdateDiagnostics()
 		if (auto const* secondary = error->secondarySourceLocation())
 			for (auto&& [secondaryMessage, secondaryLocation]: secondary->infos)
 			{
-				Json::Value jsonRelated;
+				Json jsonRelated(Json::object());
 				jsonRelated["message"] = secondaryMessage;
 				jsonRelated["location"] = toJson(secondaryLocation);
-				jsonDiag["relatedInformation"].append(jsonRelated);
+				jsonDiag["relatedInformation"].emplace_back(jsonRelated);
 			}
 
-		diagnosticsBySourceUnit[*location->sourceName].append(jsonDiag);
+		diagnosticsBySourceUnit[*location->sourceName].emplace_back(jsonDiag);
 	}
 
 	if (m_client.traceValue() != TraceValue::Off)
 	{
-		Json::Value extra;
-		extra["openFileCount"] = Json::UInt64(diagnosticsBySourceUnit.size());
+		Json extra;
```

Comment:
Again I'm wondering if you're missing a ``= Json::object()`` here or if it's redundant in most or all other places :-).

Reply:
its mostly redundant in the other places.. I'll clean that up